### PR TITLE
Website updates for glvis-4.0 release

### PR DESCRIPTION
**Pull request by tzanio**

### CODE REVIEW

File: src/about.md [hunk 1]
```diff
@@ -13,21 +13,29 @@ Please cite with:
...
-- Ilya Lashuk
+- [Ilya Lashuk](https://scholar.google.com/citations?user=5KQiAUoAAAAJ&hl=en)
 - [Robert Rieben](http://people.llnl.gov/rieben1)
 - [Joachim Schöberl](http://www.asc.tuwien.ac.at/~schoeberl/wiki/index.php/Joachim_Sch%C3%B6berl)
+- [Tom Stitt](https://github.com/tomstitt)
 - [Mark Stowell](http://people.llnl.gov/stowell1)
 - [Vladimir Tomov](http://people.llnl.gov/tomov2)
 - [Stanimire Tomov](http://icl.cs.utk.edu/~tomov)
 - [Umberto Villa](http://users.ices.utexas.edu/~uvilla/)
+- [Max Yang](https://github.com/kanye-quest)
```
Comment (tzanio) [inline src/about.md:37]: @kanye-quest -- I put your GH page as a placeholder, please feel free to replace with a different page :)

**Comment by tzanio** [comment]
@kanye-quest -- do you want to update the 

- Building (https://github.com/GLVis/web/blob/master/src/building.md, https://glvis.org/building/) 

and/or 

- Features (https://github.com/GLVis/web/blob/master/src/features.md, https://glvis.org/features/)

 pages?

**Comment by tzanio** [comment]
@GLVis/everyone -- Do you have any cool GLVis visualization images that you are willing to share in our [Gallery](https://glvis.org/gallery/) as part of the glvis-4.0 release?

**Comment by tzanio** [comment]
Related to this PR, should we update the classes we are highlighting in

http://glvis.github.io/doxygen/html/index.html

Source is here: https://github.com/GLVis/doxygen/blob/gh-pages/index.html#L60-L67

**Comment by tzanio** [comment]
Related to this PR: should we update the classes we are highlighting in the Doxygen docs?

👉  http://glvis.github.io/doxygen/html/index.html

Source is here: https://github.com/GLVis/doxygen/blob/gh-pages/index.html#L60-L67

File: src/building.md [hunk 2]
```diff
@@ -2,52 +2,75 @@
 
 A simple tutorial how to build and run GLVis together with MFEM. For more details, see the [INSTALL](https://raw.githubusercontent.com/glvis/glvis/master/INSTALL) file.
 
+## Dependencies
```
Comment (tzanio) [inline src/building.md:5]: Thanks for this update! 🙏

**Review by rw-anderson — APPROVED** [review]

**Comment by tzanio** [comment]
@v-dobrev, @acfisher and @kanye-quest -- please review

**Review by publixsubfan — APPROVED** [review]

**Pull request merged**